hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1630,6 +1630,14 @@
     <description>Byte limit for Raft's Log Worker queue.
     </description>
   </property>
+  <property>
+    <name>ozone.om.ratis.log.purge.gap</name>
+    <value>1024</value>
+    <tag>OZONE, OM, RATIS</tag>
+    <description>The minimum gap between log indices for Raft server to purge
```

Comment:
Does this mean we will snapshot every 1024 transactions?

Reply:
No, when a snapshot is being taken, if the gap between log purges is more than 1024, then it will purge the logs. Snapshot frequency is not dependent on this.

Reply:
Let's set this to a higher value. We don't need to be too aggressive about purging Ratis logs.

Reply:
1024 transactions is 100ms worth of edits in a busy cluster. We could set this as high as 1M maybe to keep more history. :)

Reply:
Agree. Will update it.

Reply:
Agree. Will update it.